cirq-google/cirq_google/ops/dynamical_decoupling_tag.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2024 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional
+
+import attrs
+
+import cirq_google.api.v2.program_pb2 as v2
+
+
+SUPPORTED_DD_PROTOCOLS = frozenset(
+    [
+        "X",  # An even number of X
+        "Y",  # An even number of Y
+        "XY4",  # Repititions of XYXY blocks.
+        "XY8",  # Reptitions of XYXYYXYX blocks.
+    ]
+)
+
+
+@attrs.frozen
+class DynamicalDecouplingTag:
+    """A tag to indicate using DD to fill qubit time."""
+
+    protocol: str = attrs.field()  # Which DD protocol to use.
```

Comment:
Nit: `attrs.field()` seems unnecessary.  Also please describe in the docstring, e.g.,
```
    Attributes:
        protocol: The name of the decoupling protocol (eg 'X', 'XY4').
```

Reply:
Added the docstring

> Nit: attrs.field() seems unnecessary. Also please describe in the docstring, e.g.,

this is requried by `attrs` for the validation code